app/assets/stylesheets/styleguide.scss

```diff
@@ -1,38 +1,50 @@
+// Colors
 .color-palette {
+  display: flex;
+  flex-direction: row;
   margin-bottom: 2em;
-  overflow: hidden;
   li {
-    display: inline-block;
-    float: left;
-    height: 50px;
+    display: flex;
+    border-radius: 3px;
+    flex-grow: 1;
```

Comment:
Another Compass mix-in for you: http://compass-style.org/reference/compass/css3/flexbox/#mixin-flex-grow